core/src/main/scala/org/apache/spark/ui/exec/ExecutorThreadDumpPage.scala

```diff
@@ -60,44 +61,49 @@ private[ui] class ExecutorThreadDumpPage(parent: ExecutorsTab) extends WebUIPage
           }
         }
       }.map { thread =>
-        val threadName = thread.threadName
-        val className = "accordion-heading " + {
-          if (threadName.contains("Executor task launch")) {
-            "executor-thread"
-          } else {
-            "non-executor-thread"
```

Comment:
We're losing the highlighting of the task threads here.  A user could search for the thread name, but I don't think users will know that is what they should search for.

I can't think of a clean UI treatment to keep this, so I'm OK removing it, but I just wanted to call it out in case others have ideas / opinions.